src/lib.rs

```diff
@@ -15,7 +15,6 @@
     trivial_casts,
     trivial_numeric_casts,
     unreachable_pub,
-    unsafe_code,
```

Comment:
Please add this back in and like I did in `src/wrapper/xsetroot.rs`, allow unsafe code just for your wrapper module. This way, no one accidentally adds unsafe code in the remaining application.